README.MD

```diff
@@ -27,11 +27,9 @@ After you're finished please remove all the comments and instructions!
 
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+ <!-- <img src="#" alt="logo" width="140"  height="auto" /> -->
```

Comment:
- [ ] As mentioned by the previouse reviewer kindly remove all placeholder text they are there just for explanation comment ,kindly remove all to make your readme file look professional.